docs/cn/core-services/Caching.md

```diff
@@ -1,259 +1,507 @@
 ---
 layout: global
-title: Alluxio存储管理
-nickname: Alluxio存储管理
+title: Caching
+nickname: Caching
 group: Core Services
-priority: 0
+priority: 1
 ---
 
-* 内容列表
+* Table of Contents
 {:toc}
 
-Alluxio管理了Alluxio workers节点的包括内存在内的本地存储，来充当分布式缓冲缓存区。这个在用户应用程序和各种底层存储之间的快速数据层可以很大程度上改善I/O性能。
+## Alluxio存储概述
 
-每个Alluxio节点管理的存储数量和类型由用户配置决定。Alluxio还支持层次化存储，这使得系统存储能够感知介质，让数据存储获得类似于L1/L2 cpu缓存的优化。
+本文档的目的是向用户介绍Alluxio存储和
+在Alluxio存储空间中可以执行的操作背后的概念。 与元数据相关的操作
+例如同步和名称空间，请参阅
+[有关命名空间管理的页面]
+({{ '/en/core-services/Unified-Namespace.html' | relativize_url }})
+
+Alluxio在帮助统一跨各种平台用户数据的同时还有助于为用户提升总体I / O吞吐量。 
+Alluxio是通过把存储分为两个不同的类别来实现这一目标的。
+
+-** UFS(底层文件存储，也称为底层存储)**
```

Comment:
remember to add a space after the `-`, this  doc format doesn't work if no spaces following
do the same for all the following lines start with `-`